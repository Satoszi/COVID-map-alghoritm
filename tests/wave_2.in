8 8 3 15
wave_2
++..++..
||++||++
||||||||
||||||||
||||||||
||||||||
|||++|||
|++..++.