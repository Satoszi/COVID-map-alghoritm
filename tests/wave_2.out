6 7
2 7
0 7
7 6
3 6
5 4
1 4
6 3
4 3
2 3
0 3
7 2
3 2
5 0
1 0
